Lib/fontTools/varLib/instancer/__init__.py

```diff
@@ -1413,16 +1421,41 @@ def parseArgs(args):
     if len(axisLimits) != len(options.locargs):
         parser.error("Specified multiple limits for the same axis")
 
+    if axisLimits and options.named_instances:
+        parser.error("Specified limits and --fvar")
+
     return (infile, axisLimits, options)
 
 
 def main(args=None):
     """Partially instantiate a variable font."""
     infile, axisLimits, options = parseArgs(args)
-    log.info("Restricting axes: %s", axisLimits)
 
     log.info("Loading variable font")
     varfont = TTFont(infile)
+    if options.named_instances:
+        for instance in varfont["fvar"].instances:
+            name = varfont["name"].getDebugName(instance.subfamilyNameID)
+            if not re.search(options.named_instances, name):
+                continue
+            axisLimits = instance.coordinates
+            log.info("Restricting axes: %s (%s)", axisLimits, name)
```

Comment:
in this context, we use "restricting an axis" to mean level-3 instancing (i.e. modify min/max), as opposed to `pin an axis` for level-1 (pin at default) and 2 (pin at arbitrary location). If you still wanna log, keep it generic, like "Instancing axes: %s"